proposals/nnnn-move-function.md

```diff
@@ -0,0 +1,455 @@
+# Move Function + "Use After Move" Diagnostic
+
+* Proposal: [SE-NNNN](nnnn-move-function.md)
+* Authors: [Michael Gottesman](https://github.com/gottesmm), [Andrew Trick](https://github.com/atrick)
+* Review Manager: TBD
+* Status: Implemented on main as stdlib SPI (`_move` instead of `move`)
+
+<!--
+*During the review process, add the following fields as needed:*
+
+* Implementation: [apple/swift#NNNNN](https://github.com/apple/swift/pull/NNNNN) or [apple/swift-evolution-staging#NNNNN](https://github.com/apple/swift-evolution-staging/pull/NNNNN)
+* Decision Notes: [Rationale](https://forums.swift.org/), [Additional Commentary](https://forums.swift.org/)
+* Bugs: [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN), [SR-MMMM](https://bugs.swift.org/browse/SR-MMMM)
+* Previous Revision: [1](https://github.com/apple/swift-evolution/blob/...commit-ID.../proposals/NNNN-filename.md)
+* Previous Proposal: [SE-XXXX](XXXX-filename.md)
+-->
+
+## Introduction
+
+In this document, we propose adding a new function called `move` to the swift
+standard library, which ends the lifetime of a specific local `let`,
+local `var`, or `consuming` parameter. In order to enforce this, the compiler will
```

Comment:
Overall suggestion for making markup documents easier to review:  Put a line break after every full stop.

I suggest you add a sentence here after "parameter."
```
This gives developers a way to control retain/release operations and avoid
unnecessary copy-on-write behaviors in performance-critical code.
```
Then this paragraph will provide a good summary:  A sentence saying _what_ you propose, a sentence about _why_, and a sentence summarizing _how_ the compiler will support this operation.